bci_essentials/bci_data.py

```diff
@@ -1061,27 +1083,27 @@ def package_resting_state_data(self):
 
     def main(
         self,
-        buffer=0.01,
-        eeg_start=0,
-        max_channels=64,
-        max_samples=2560,
-        max_windows=1000,
-        max_loops=1000000,
-        training=True,
-        online=True,
-        train_complete=False,
-        iterative_training=False,
-        live_update=False,
-        print_markers=True,
-        print_training=True,
-        print_fit=True,
-        print_performance=True,
-        print_predict=True,
-        pp_type="bandpass",  # preprocessing method
-        pp_low=1,  # bandpass lower cutoff
-        pp_high=40,  # bandpass upper cutoff
-        pp_order=5,  # bandpass order
-    ):
+        buffer: float = 0.01,
+        eeg_start: int = 0,
+        max_channels: int = 64,
+        max_samples: int = 2560,
+        max_windows: int = 1000,
+        max_loops: int = 1000000,
+        training: bool = True,
+        online: bool = True,
+        train_complete: bool = False,
+        iterative_training: bool = False,
+        live_update: bool = False,
+        print_markers: bool = True,
+        print_training: bool = True,
+        print_fit: bool = True,
+        print_performance: bool = True,
+        print_predict: bool = True,
+        pp_type: str = "bandpass",  # preprocessing method
+        pp_low: int = 1,  # bandpass lower cutoff
```

Comment:
pp_low and pp_high need to be floats, because they are the preprocessing filter cutoffs and in many cases we may want a cutoff between 0 and 1.

Reply:
This goes for other places pp_low and pp_high might be too.

Reply:
Ok, I'll update the docs to reflect this. Thanks!

Reply:
Solved in the [newest commit](https://github.com/kirtonBCIlab/bci-essentials-python/pull/64/commits/877bca154b2b02de83e3389cc9c511c461739c0b). Some bci_data.py preprocessing() arguments had to be updated as well to make this work. 

Resolving the issue now